webapp/channels/src/components/profile_popover/profile_popover_custom_attributes.tsx

```diff
@@ -17,42 +17,43 @@ const ProfilePopoverCustomAttributes = ({
     userID,
 }: Props) => {
     const dispatch = useDispatch();
-    const [customAttributeValues, setCustomAttributeValues] = useState<Record<string, string>>({});
+    const userProfile = useSelector((state: GlobalState) => getUser(state, userID));
     const customProfileAttributeFields = useSelector((state: GlobalState) => getCustomProfileAttributes(state));
 
     useEffect(() => {
-        const fetchValues = async () => {
-            const response = await Client4.getUserCustomProfileAttributesValues(userID);
-            setCustomAttributeValues(response);
-        };
-        dispatch(getCustomProfileAttributeFields());
-        fetchValues();
-    }, [userID, dispatch]);
-    const attributeSections = Object.values(customProfileAttributeFields).map((attribute) => {
-        const value = customAttributeValues[attribute.id];
-        if (!value) {
-            return null;
+        if (!userProfile.custom_profile_attributes) {
+            dispatch(getCustomProfileAttributeValues(userID));
```

Comment:
This can be removed when initial profile is retrieved with custom profille attributes. Currently, it will only be called once.